client/src/includes/initSkipLink.test.js

```diff
@@ -0,0 +1,25 @@
+import { Application } from '@hotwired/stimulus';
+
```

Comment:
This test needs work - the Stimulus application import will not work (I know this was pulled from future proposed code but please revisit). 

Also the test itself should test what happens when you click the button. Not just the initial state where there are no focused elements.

Reply:
Probably the reason the test i wrote for the button kept failing, i will redo thanks